### PR TITLE
Add pathway analysis DB versions (3)

### PR DESCRIPTION
**Pull request by WackerO**
This PR adds the versions of the databases (KEGG, REAC) used in the pathway analysis to the report as mentioned in #72 like this:
`
For pathway analysis, the R packages gprofiler2 v0.2.1, AnnotationDbi v1.56.1 and org.Mm.eg.db v1.56.1 were employed. Also, the following databases were queried: KEGG: KEGG FTP Release 2022-05-16, REAC: annotations: BioMart, classes: 2022-5-18.`

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnadeseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnadeseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: tests/test_batcheffect.yml [hunk 1]
```diff
@@ -4,15 +4,15 @@
     - test_batcheffect
   files:
     - path: results_test/differential_gene_expression/DE_genes_tables/DE_contrast_batch_b2_vs_b1.tsv
-      md5sum: 4c8bcede761f82803c262f28a13eb5ce
+    #      md5sum: 4c8bcede761f82803c262f28a13eb5ce
```
Comment (SusiJo) [inline tests/test_batcheffect.yml:7]: Why were they uncommented? Are the files suddenly not producing the same md5sums anymore?

File: assets/RNAseq_report.Rmd [hunk 2]
```diff
@@ -1775,12 +1774,18 @@ An aggregation of the quality control for the RNAseq analysis was performed with
...
 name_species       = params$species_library
 version_species    = packageVersion(name_species)
-cat(paste("For pathway analysis, the R packages `gprofiler2 v",version_gprofiler2,"`, `AnnotationDbi v", version_annotation, "` and `", name_species, " v", version_annotation, "` were employed.", sep=''))
+database_string = ""
+database_version_info <- get_version_info(params$organism)
+for (db in datasources) {
+    database_string <- paste0(database_string, "`", db, ": ", gsub("\n", ", ", database_version_info$sources[[db]]$version), "`, ")
+}
+database_string <- paste0(substr(database_string,1,nchar(database_string)-2), ".")
+cat(paste0("For pathway analysis, the R packages `gprofiler2 v",version_gprofiler2,"`, `AnnotationDbi v", version_annotation, "` and `", name_species, " v", version_annotation, "` were employed. Also, the following databases were queried: ", database_string))
```
Comment (SusiJo) [inline assets/RNAseq_report.Rmd:1788]:
```suggestion
cat(paste0("For pathway analysis, the R packages `gprofiler2 v", version_gprofiler2," `, `AnnotationDbi v", version_annotation, "` and `", name_species, " v", version_annotation, "` were employed. Also, the following databases were queried: ", database_string))
```

Comment (SusiJo) [reply assets/RNAseq_report.Rmd:1788]: You could also just use `paste` and leave out the whitespaces.

File: tests/test_batcheffect.yml (hunk 1 above)
Comment (WackerO) [reply tests/test_batcheffect.yml:7]: Yep...they changed from time to time, I am not sure when exactly this happens, I believe in different PRs the 5sums change, almost as if the github server runner changes the files...

Comment (SusiJo) [reply tests/test_batcheffect.yml:7]: How do the md5sums behave when you test locally? Are they stable over multiple runs?

Comment (WackerO) [reply tests/test_batcheffect.yml:7]: Testing this right now

Comment (WackerO) [reply tests/test_batcheffect.yml:7]: I cannot make head nor tail of that...On another branch, the 5sums are still enabled and it works. Locally, I got different sums when running the same profile on cfc and thanos. I'll reenable the 5sums here. If they again don't work, I'll probably have to delete them...

**Review by ggabernet — APPROVED** [review]
Looks good to me and the md5sums seem to be fine now

**Pull request merged**